addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -587,7 +587,10 @@ export class PosStore extends Reactive {
             order.select_orderline(order.get_last_orderline());
         }
 
+        // FIXME: Put this in an effect so that we don't have to call it manually.
```

Comment:
We can try to run it with the effect in data_service. But this methods is only called twice.
- In addLineToCurrentOrder
- and syncAllOrders